roles/kibana/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+kibana_package_name: kibana
+kibana_service_name: kibana
+kibana_config_dir: /etc/kibana
+kibana_config_file: '{{ kibana_config_dir }}/kibana.yml'
+kibana_config_mode: 0644
+kibana_owner: kibana
+kibana_group: kibana
+
+# "0.0.0.0" to allow public access
+# "localhost" to allow access through proxy only
+kibana_server_bind: localhost
+kibana_server_port: 5601
+kibana_server_elasticsearch_url: 'http://localhost:9200'
```

Comment:
That's a good point. I'm not sure if it is possible to "borrow" variables from other roles without explicitly depending on it ... on the other hand having kibana role be dependent on elasticsearch role makes sense ... ok, once elasticsearch is merged I will refactor it.